CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+
+## 1.7.4
+  * Request Timeout Implementation
```

Comment:
Can you include the PR that implemented this?

Reply:
Done